tests/Mock/AdapterMock.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file is part of VehicleManagement
+ * Date: 8/13/15 - 11:34 PM
+ * (c) Sixt GmbH & Co. Autovermietung KG
```

Comment:
Good catch. Crazy, my phpstorm project class template switch did not have a rule for the event-store project. Fortunately, only the two new added class files have the wrong header. I'll change that and align my template set up.